contracts/multisig/src/testdata/query_multisig_session_returns_session.golden

```diff
@@ -32,14 +32,14 @@
     "created_at": 0
   },
   "signatures": {
-    "signer2": {
-      "ecdsa": "a7ec5d1c15e84ba4b5da23fee49d77c5c81b3b1859411d1ef8193bf5a39783c76813e4cf4e1e1bfa0ea19c9f5b61d25ce978da137f3adb1730cba3d842702e72"
+    "signer1": {
```

Comment:
non determinism in the test, let's sort it first?

Reply:
can't sort a hashmap. I just reverted this file, can we revisit this at another time? This is not really important at the moment, and just want to get the ITS stuff done.